content/patterns/table/examples/sortable-table.html

```diff
@@ -65,7 +65,7 @@ <h2 id="ex_label">Example</h2>
           <table class="sortable">
             <caption>
               Students currently enrolled in WAI-ARIA 101
-              <span class="sr-only">, column headers with buttons are sortable.</span>
+              <span class="sr-only">(column headers with buttons are sortable).</span>
```

Comment:
Important to have a space before the opening parend. Without the space, the there is no separation between the visible text and the off-screen text.

```suggestion
              <span class="sr-only"> (column headers with buttons are sortable).</span>
```